GazePianoPrototype/App.xaml

```diff
@@ -48,6 +48,8 @@
                 <Setter Property="HorizontalAlignment" Value="Center"/>
                 <Setter Property="VerticalAlignment" Value="Center"/>
             </Style>
+
+            <x:String x:Key="pianoFunctionsDwell">00:00:00.75</x:String>
```

Comment:
(For future changes) I wonder if we could add the DwellDuration/FixationDuration directly to styles in the resourcedictionary